geonode_mapstore_client/client/js/components/Menu/MenuItem.js

```diff
@@ -70,7 +70,7 @@ const MenuItem = ({ item, menuItemsProps, containerNode, tabIndex, classItem = '
 
     if (type === 'link') {
         return (
-            <NavLink href={href} target={target} className={btnClassName}>{labelId && <Message msgId={labelId} /> || label}</NavLink>
+            <NavLink href={href} target={target} className={btnClassName}>{labelId && <Message msgId={labelId} /> || label}{item.datasetEditor && resourceName}</NavLink>
```

Comment:
We are not sure what is the best position of resourceName based on the language in use for this reason I propose to remove `{item.datasetEditor && resourceName}` and instead use the msgParams

`<Message msgId={labelId} msgParams={{ resourceName }}/>`

then inside a translation you can use: `< Go back to {{ resourceName }}`

This change will remove also `item.datasetEditor` that's too specific to one use case